### ISSUE TITLE
Feat Request: Ability to Set Height to "Auto"

### ISSUE THREAD

**Issue by actuallyReallyAlex** [typicode/tlapse#17]
I know that you can set the width and height by specifying them as options, but if you don't already know the height of the page, it would be nice to add an option to have that calculated for you.

Something like `tlapse localhost:3000 -v 1300 x auto` or similar.

**Comment by typicode** [comment]
Did a quick search in https://github.com/GoogleChrome/puppeteer/blob/v1.10.0/docs/api.md but couldn't find anything. If there is actually, yes, why not, PR welcome :)

I wonder also if having screenshots with different height wouldn't make generating GIFs more complicated?

**Comment by actuallyReallyAlex** [comment]
Yeah, you might be right about it being more complicated. Right now I just have the height to some ridiculous number, so that I can just go and crop it down later.
